API/Controllers/LessonsController.cs

```diff
@@ -0,0 +1,30 @@
+using API.Data;
+using API.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class LessonsController : ControllerBase
+    {
+        public CourseContext Context { get; }
+        public LessonsController(CourseContext context)
+        {
+            Context = context;
+        }
+
+        [HttpGet]
+        public ActionResult<List<Lesson>> GetLessons()
+        {
+            var lessons = Context.Lessons.ToList();
+            return Ok(lessons);
+        }
+
+        [HttpGet("{id}")]
+        public ActionResult<Lesson> GetLesson(int id)
+        {
+            return Context.Lessons.Find(id);
+        }
```

Comment:
It is better to make this controller in asynchronous format

Reply:
Yes, that's great idea.